scripts/py_matter_yamltests/matter_yamltests/pseudo_clusters/clusters/delay_commands.py

```diff
@@ -58,6 +58,9 @@ async def WaitForMs(self, request):
                 duration_in_ms = argument['value']
 
         sys.stdout.flush()
+        if type(duration_in_ms) is str:
```

Comment:
nit: int seems always safe, so we could just do:

```
time.sleep(int(duration_in_ms) /  1000)
```

and not do the typecheck